examples/resources/kentik-synthetic_test-bulk-create/one_agent_per_country/resource.tf

```diff
@@ -0,0 +1,53 @@
+// Create a test with list of agents - one agent per country.
+
+data "kentik-synthetics_agents" "agents" {}
+
+locals {
+  country_to_ids_map = {for agent in data.kentik-synthetics_agents.agents.items: agent.country => agent.id...}
+  agent_ids = [for key, val in local.country_to_ids_map: val[0]]
```

Comment:
Is it idempotent? Are agent IDs for given country always in the same order?

Reply:
The order in which agents are returned by the API is undefined. I did a quick test and in 2 successive requests 2 agents were swapped in the listing (the overall set was the same). We will need to sort agents by id to make sure that the selected set remains the same as long as all previously selected agents are still available.

Reply:
Addressed: 794ec2530695564cfc850b0d8d4722b3b336a33c

Reply:
On `64ec7a1` TF apply returns following error:
```bash

│ Error: Unsupported attribute
│ 
│   on resource.tf line 55, in output "one_agent_per_country-test-output":
│   55:   value = sort(data.kentik-synthetics_agents.agents.items.id)
│ 
│ Can't access attributes on a list of objects. Did you mean to access attribute "id" for a specific element of the list, or across all elements of the list?
```

Reply:
I've pushed quick fix

Reply:
Now I have 
```
kentik-synthetics_test.one_agent_per_country-test: Creating...
╷
│ Error: resourceDataToTestHealthSettings: invalid dataSlice[0] type, got: <nil>, want: map[string]interface{}
│ 
│   with kentik-synthetics_test.one_agent_per_country-test,
│   on resource.tf line 13, in resource "kentik-synthetics_test" "one_agent_per_country-test":
│   13: resource "kentik-synthetics_test" "one_agent_per_country-test" {
```
I use Terraform v1.0.3 and `run.sh` script (which builds latest provider)